Sources/NIOCore/Utilities.swift

```diff
@@ -269,3 +270,30 @@ extension System {
         return nil
     }
 }
+
+#if os(Windows)
+@usableFromInline
+func windows_strerror(_ errnoCode: CInt) -> String? {
+    withUnsafeTemporaryAllocation(of: CChar.self, capacity: 256) { ptr in
+        if strerror_s(ptr.baseAddress, ptr.count, errnoCode) == 0 {
+            return String(cString: UnsafePointer(ptr.baseAddress!))
+        }
+        return nil
+    }
+}
+
+func windows_getenv(_ env: String) -> String? {
```

Comment:
These are Swift functions, we can use a Swift naming convention for them.

Reply:
You mean with a `Windows` namespace around them? Happy to do that!